datasets/brandkranen/dataset.json

```diff
@@ -0,0 +1,91 @@
+{
+    "id": "brandkranen",
+    "type": "dataset",
+    "auth": "FP/MDW",
+    "authorizationGrantor": "n.v.t.",
+    "theme": [
+        "Veiligheid, wonen"
+    ],
+    "homepage": "https://data.amsterdam.nl",
+    "owner": "Waternet",
+    "dateModified": "2021-01-01",
+    "spatialDescription": "De stad Amsterdam",
+    "version": "1.0.0",
+    "title": "Brandkranen",
+    "language": "nl",
+    "dateCreated": "2021-01-01",
+    "license": "Creative Commons, Naamsvermelding",
+    "hasBeginning": "",
+    "hasEnd": "",
+    "accrualPeriodicity": "eenmalig",
+    "publisher": "OIS",
+    "description": "De dataset bevat de openbare brandkranen in Amsterdam.",
+    "status": "beschikbaar",
+    "keywords": [
+        "brandweer",
+        "brandkranen"
+    ],
+    "crs": "EPSG:28992",
+    "objective": " ",
+    "temporalUnit": "",
+    "contactPoint": {
+        "name": "",
+        "email": "datadiensten.ois@amsterdam.nl"
+    },
+    "tables": [
+        {
+            "id": "brandkranen",
+            "type": "table",
+            "version": "1.0.0",
+            "description": "De dataset bevat de openbare brandkranen in Amsterdam.",
+            "schema": {
+                "$schema": "http://json-schema.org/draft-07/schema#",
+                "type": "object",
+                "additionalProperties": false,
+                "identifier": "id",
+                "required": [
+                    "id",
+                    "schema"
+                ],
+                "mainGeometry": "geometrie",
+                "display": "id",
+                "properties": {
+                    "schema": {
+                        "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+                    },
+                    "id": {
```

Comment:
Het toevoegen van de `title` en `description` attributes op de velden is [best-practice](https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#permitted-attributes), hoewel dit geen verplichte velden zijn. Zou je die attributes aan alle velden kunnen toevoegen?

Reply:
Deze metadata attributen vullen we vooral omdat ze onderdeel van DCAT datamodel maken. Ooit zullen  velden hier automatisch gevuld gaat worden (dit is tenminste wat het ons is verteld is) 
https://data.amsterdam.nl/datasets/zoek/